tests/ui/specs/Input/Agate-Input-Page.js

```diff
@@ -11,9 +11,24 @@ class AgateInputPage extends Page {
 		super.open('Agate-Input-View', urlExtra);
```

Comment:
You don't need to use'-' for Page file name.
In the PLAT-126330, there was not.
Agate-PanelPage.js is Better.
In fact, I don't think it's good to prefix of "Agate-".
"-Spec" and "-View" is mandatory to filter on ui-test-utils. but Page is not. just use camelcase.

Reply:
I see. Thank you!